libdivide.h

```diff
@@ -1234,8 +1658,24 @@ int64_t libdivide_s64_branchfree_recover(const struct libdivide_s64_branchfree_t
     return libdivide_s64_recover((const struct libdivide_s64_t *)denom);
 }
 
+// Simplest possible vector type division: treat the vector type as an array
+// of underlying native type.
+#define SIMPLE_VECTOR_DIVISION(IntT, VecT, Algo) \
+    static const size_t count = sizeof(VecT) / sizeof(IntT); \
```

Comment:
Please remove the `static` here, it's probably harmless but it's possible it will be emitting weak symbols which we don't want. `const size_t count = sizeof(VecT) / sizeof(IntT);` is fine.